src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnection.cs

```diff
@@ -237,14 +294,19 @@ static internal List<string> GetColumnEncryptionSystemKeyStoreProviders()
         /// This function returns a list of custom provider dictionary currently registered.
         /// </summary>
         /// <returns>Combined list of provider names</returns>
-        static internal List<string> GetColumnEncryptionCustomKeyStoreProviders()
+        internal static List<string> GetColumnEncryptionCustomKeyStoreProviders(SqlConnection connection)
```

Comment:
Please update description to add param and note that same method also returns "Global" list if connection is null.
Same in netcore.

Reply:
Done.